docs/development/deploying-at-scale/index.md

```diff
@@ -1,3 +1,37 @@
 # Deploying at scale
 
 REANA can be easily deployed on large Kubernetes clusters using Helm. Useful for production instances.
+
+## Pre-requisites
+
+Create a shared volume:
+
+```console
+
+```
+
+## Deploy
+
+- (1) Add REANA chart repository:
+```console
+$ helm repo add reanahub https://reanahub.github.io/reana
+"reanahub" has been added to your repositories
+$ helm repo update
+Hang tight while we grab the latest from your chart repositories...
+...Successfully got an update from the "reanahub" chart repository
+...Successfully got an update from the "cern" chart repository
+...Successfully got an update from the "stable" chart repository
+Update Complete. ⎈ Happy Helming!⎈
+```
+- (2) Deploy REANA
+```console
+$ helm install my-awesome-reana reanahub/reana --wait
```

Comment:
Depends on REANA Helm chart release.

Reply:
Solved in https://github.com/reanahub/reana/pull/287, ready to be merged CC @tiborsimko.